packages/salesforcedx-vscode-core/package.json

```diff
@@ -364,9 +352,49 @@
           "command": "sfdx.force.source.delete.current.file",
           "when": "sfdx:project_opened && sfdx:default_username_has_no_change_tracking && resourceLangId != 'forcesourcemanifest' && editorIsOpen"
         },
+        {
+          "command": "sfdx.force.set.default.org",
+          "when": "sfdx:project_opened"
+        },
         {
           "command": "sfdx.force.lightning.lwc.create",
           "when": "sfdx:project_opened"
+        },
+        {
+          "command": "sfdx.force.lightning.lwc.create",
+          "when": "sfdx:internal_dev"
```

Comment:
Consolidated all the lightning create commands here. Unfortunately, vscode's when clause doesn't support the use of OR statements so I had to duplicate the command entries.